antora-ui-camel/src/css/footer.css

```
@@ -64,7 +65,10 @@ footer .footer dl dd {
 
 .edit {
   max-width: var(--static-max-width--desktop);
-  margin: auto;
+  position: relative;
+  margin-top: 10em;
```

Perhaps we could use `rem` here?

```suggestion
  margin-top: 10rem;
```